RegisterController.js

```diff
@@ -6,6 +6,12 @@
 import {AccountService} from './AccountService.js';
 import pDebounce from 'p-debounce';
 
+/**
+ * encapsulates the registration methods
+ * @param {Object} config
+ * @param {Number} config.debounceExists - defaults to 500
```

Comment:
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

Reply:
- [ ] change to an optional params using = after the type